enqueue.go

```diff
@@ -64,6 +64,48 @@ func (e *Enqueuer) Enqueue(jobName string, args map[string]interface{}) (*Job, e
 	return job, nil
 }
 
+// EnqueueChain will enqueue the specified jobs in a chain one-by-one fashion.
+// Example: e.EnqueueChain(work.C{"job1": work.Q{"arg": "foo"}, "job2": work.Q{"arg": "bar"}})
+func (e *Enqueuer) EnqueueChain(jobs C) (*Job, error) {
+	var jobList []*Job
+
+	for _, v := range jobs {
+		for i, j := range v {
+			job := &Job{
+				Name:       i,
+				ID:         makeIdentifier(),
+				EnqueuedAt: nowEpochSeconds(),
+				Args:       j,
+			}
+			jobList = append(jobList, job)
+		}
+	}
+
+	for i := len(jobList) - 1; i > 0; i-- {
+		if i > 0 {
+			jobList[i-1].OnSuccess = jobList[i]
+		}
+	}
+
+	rawJSON, err := jobList[0].serialize()
+	if err != nil {
+		return nil, err
+	}
+
+	conn := e.Pool.Get()
+	defer conn.Close()
+
+	if _, err := conn.Do("LPUSH", e.queuePrefix+jobList[0].Name, rawJSON); err != nil {
+		return nil, err
+	}
+
+	if err := e.addToKnownJobs(conn, jobList[0].Name); err != nil {
```

Comment:
You should loop over the job chain and add all jobs to the set of known jobs here.